napari/layers/utils/layer_utils.py

```diff
@@ -751,37 +900,10 @@ def _get_default_column(column: pd.Series) -> pd.Series:
 
 
 def _validate_features(
-    features: Union[Dict[str, np.ndarray], pd.DataFrame],
+    features: Optional[Union[Dict[str, np.ndarray], pd.DataFrame]],
     *,
     num_data: Optional[int] = None,
 ) -> pd.DataFrame:
-    """Validates and coerces a features table into a pandas DataFrame.
```

Comment:
Why remove this docstring? I like docstrings even for private methods.

Reply:
Yes I'd be into that. Ideally, use proper sphinx links?

```
"""Validate blah blah blah.

See Also
--------
:meth:`_FeatureManager.validate`
```

Reply:
Done.